libnetwork/driver_plugin.py

```diff
@@ -70,13 +96,131 @@ def make_json_error(ex):
 # The API calls below are documented at
 # https://github.com/docker/libnetwork/blob/master/docs/remote.md
 
+# <-- Plugin activation, we activate both libnetwork and ipam plugins -->
 
 @app.route('/Plugin.Activate', methods=['POST'])
 def activate():
-    json_response = {"Implements": ["NetworkDriver"]}
+    json_response = {"Implements": ["NetworkDriver", "IpamDriver"]}
     app.logger.debug("Activate response JSON=%s", json_response)
     return jsonify(json_response)
 
+# <-- IPAM plugin API -->
+
+@app.route('/IpamDriver.GetDefaultAddressSpaces', methods=['POST'])
+def get_default_address_spaces():
+    json_data = request.get_json(force=True)
+    app.logger.debug("GetDefaultAddressSpace JSON=%s", json_data)
+    json_response = {
+        "LocalDefaultAddressSpace": LOCAL_CIDR_ADDR_SP,
```

Comment:
I'm not the biggest fan of having constants defined somewhere else when 
- they're only used once
- it's only a small program.

It's just bloating the code and making the reader jump around unnecessarily.